Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -298,7 +292,28 @@ -(void)filterContentForSearchText:(NSString*)searchText scope:(NSString*)scope {
     NSPredicate *predicate = [NSPredicate predicateWithFormat:@"(SELF.title LIKE[cd] %@)", searchWithWildcards];
     
     self.filteredEventsArray = [NSMutableArray arrayWithArray:[self.allEvents filteredArrayUsingPredicate:predicate]];
-    NSLog(@"fileteredArr: %@" , self.filteredEventsArray);
+
+    NSMutableDictionary *searchEvents = [[NSMutableDictionary alloc] init];
+    for (GAEvent *event in self.filteredEventsArray) {
+        NSString *eventDate = event.date;
+     
+        if ( searchEvents[eventDate] ) {
+            [searchEvents[eventDate] addObject:event];
```

Comment:
This line is the same as in the else clause. Why not bump it outside of the conditional?